dig.go

```diff
@@ -228,6 +231,7 @@ func New(opts ...Option) *Container {
 		values:    make(map[key]reflect.Value),
 		groups:    make(map[key][]reflect.Value),
 		rand:      rand.New(rand.NewSource(time.Now().UnixNano())),
+		dry:       false,
```

Comment:
Nit, but i'm not sure it adds much to explicitly set `dry` to false on instantiation.